nixos/tests/all-tests.nix

```
@@ -1205,6 +1205,7 @@ in
   shadowsocks = handleTest ./shadowsocks { };
   shattered-pixel-dungeon = handleTest ./shattered-pixel-dungeon.nix { };
   shiori = handleTest ./shiori.nix { };
+  shoko = handleTest ./shoko.nix {};
```

nixfmt-check is failing, to fix:
```suggestion
  shoko = handleTest ./shoko.nix { };
```